src/main/java/pojlib/util/JREUtils.java

```diff
@@ -197,7 +197,10 @@ public static int launchJavaVM(final Activity activity, final List<String> JVMAr
 
         userArgs.add("-XX:+UseZGC");
         userArgs.add("-XX:+ZGenerational");
-
+        userArgs.add("-XX:+UnlockExperimentalVMOptions");
+        userArgs.add("-XX:+UnlockDiagnosticVMOptions");
```

Comment:
Is this a debug-only option or is it actually used for perf?

Reply:
its used for perf, its to add jvm options

Reply:
and overall perf